lib/k6-utils/1.0.0/index.js

```diff
@@ -18,4 +18,17 @@ export function randomString(length) {
   let res = '';
   while (length--) res += charset[Math.random() * charset.length | 0];
   return res;
+}
+
+function findBetween(content, left, right) {
```

Comment:
```suggestion
export function findBetween(content, left, right) {
```

it needs to be exported in order to be usable